integr8ly/integreatly-operator
# MGDAPI-4145 - consume 3scale v0.8.4 index

# Issue link
[MGDAPI-4145](https://issues.redhat.com/browse/MGDAPI-4145)

# What
- Consume 3scale-index v0.8.4
- Add the manifests used for the release process

# Verification steps
- Verify install / uninstall rhoam
- Test upgrading rhoam from 1.22 -> 1.23 (this PR) results in 3scale operator upgrade - use workload webapp to check for downtime

I have created the indices for upgrade testing, available here:
```
quay.io/acatterm/managed-api-service-index:1.22.0
quay.io/acatterm/managed-api-service-index:1.23.0
```
